source/expressions.tex

```diff
@@ -6080,7 +6080,7 @@
 inheritance relationship between the two classes.
 \begin{note}
 For example, if the classes were defined later in the translation unit,
-a multi-pass compiler would be permitted to interpret a cast between
+a multi-pass compiler could conformingly interpret a cast between
```

Comment:
```suggestion
a multi-pass compiler could validly interpret a cast between
```

The word "conformingly" seems clunky to me (it's not even in my dictionary).

Reply:
OK, sure!